docs/rst/developer_manual/installation/sources/linux.rst

```diff
@@ -4,6 +4,306 @@
+
+Dependencies
+------------
+
+*eProsima DDS Visualizer Plugin* has the following dependencies, when installed from sources in a Linux environment:
+
+* :ref:`asiotinyxml2_sl`
+* :ref:`openssl_sl`
+* :ref:`eprosima_dependencies`
+* :ref:`qt_installation`
+
+.. _asiotinyxml2_sl:
+
+Asio and TinyXML2 libraries
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Asio is a cross-platform C++ library for network and low-level I/O programming, which provides a consistent
+asynchronous model.
+TinyXML2 is a simple, small and efficient C++ XML parser.
+Install these libraries using the package manager of the appropriate Linux distribution.
+For example, on Ubuntu use the command:
+
+.. code-block:: bash
+
+    sudo apt install libasio-dev libtinyxml2-dev
+
+.. _openssl_sl:
+
+OpenSSL
+^^^^^^^
+
+OpenSSL is a robust toolkit for the TLS and SSL protocols and a general-purpose cryptography library.
+Install OpenSSL_ using the package manager of the appropriate Linux distribution.
+For example, on Ubuntu use the command:
+
+.. code-block:: bash
+
+   sudo apt install libssl-dev
+
+.. _eprosima_dependencies:
+
+eProsima dependencies
+^^^^^^^^^^^^^^^^^^^^^
+
+If it already exists in the system an installation of *Fast DDS* library with version greater than `2.7.0`,
+just source this library when building the *DDS Visualizer Plugin* by using the command:
+
+.. code-block:: bash
+
+    source <fastdds-installation-path>/install/setup.bash
+
+In other case, just download *Fast DDS* project from sources and build it together with *DDS Visualizer Plugin* using colcon
+as it is explained in section :ref:`colcon_installation`.
+
+
+.. _qt_installation:
+
+Qt 5.15
+^^^^^^^
+
+Qt 5.15 is needed in order to build *DDS Visualizer Plugin*.
+To install this Qt version, refer to `Qt Downloads <https://www.qt.io/download>`_ website.
+
+.. note::
+
+    When going through the installation steps, make sure the box of component *Qt Charts* is checked.
+
+
+.. _colcon_installation:
+
+Colcon installation
+===================
+
+#.  Create a :code:`DDS-Visualizer-Plugin` directory and download the :code:`.repos` file that will be used to install
+    *eProsima DDS Visualizer Plugin* and its dependencies:
+
+    .. code-block:: bash
+
+        mkdir -p ~/DDS-Visualizer-Plugin/src
+        cd ~/DDS-Visualizer-Plugin
+        wget https://raw.githubusercontent.com/eProsima/plotjuggler-fastdds-plugins/main/plotjuggler_fastdds.repos
+        vcs import src < plotjuggler_fastdds.repos
+
+    .. note::
+
+        In case there is an already *Fast DDS* installation in the system it is not required to download and build
+        every dependency in the :code:`.repos` file.
+        It is just needed to download and build the *DDS Visualizer Plugin* project having sourced its dependencies.
+        Refer to section :ref:`eprosima_dependencies` in order to check how to source *Fast DDS*.
+
+    In order to build the project, it must be specified the path to the Qt 5.15 :code:`gcc_64` installation path.
```

Comment:
This is not required in this project

Reply:
The whole Qt path installation is not required. Remove these lines and remove references to `DQT_PATH`